libs/core/langchain_core/chat_history.py

```diff
@@ -154,6 +188,22 @@ def add_message(self, message: BaseMessage) -> None:
                 "Please implement add_message or add_messages."
             )
 
+    async def aadd_message(self, message: BaseMessage) -> None:
```

Comment:
Not sure about this... I'd rather discourage use of non batched variants in chat history -- the non batched variant is a design mistake since it encourages extra latency due to round-trips

Reply:
I agree. Done